mutation/mutation_partition.cc

```diff
@@ -2220,7 +2220,7 @@ stop_iteration reconcilable_result_builder::consume(clustering_row&& cr, row_tom
     }
     _live_rows += is_alive;
     auto stop = _memory_accounter.update_and_check(cr.memory_usage(_schema));
```

Comment:
> For 1M dead rows, freezing takes 130ms, unfreezing takes 500ms.

Note, this means 130ns/row to freeze, 500ns to unfreeze. At 2GHz  / 2 IPC this is 520 instructions to freeze, 2000 instructions to unfreeze. That's ridiculously high for a row that only contains a tombstone. @michoecho